src/app/components/toggle-switch/index.html

```diff
@@ -0,0 +1,77 @@
+<sky-demo-page
+  pageTitle="Toggle switch"
+  navTitle="Toggle switch"
+  skyuxModule="SkyToggleSwitchModule"
+  packageName="@skyux/forms"
+>
+
+  <sky-demo-page-summary>
+    <p>
+      The toggle switch component is useful when a value can either be "on" or "off". When users interact with a toggle switch, its value is driven through an <sky-code>ngModel</sky-code> attribute that you specify on the <sky-code>sky-toggle-switch</sky-code> element.
+    </p>
+  </sky-demo-page-summary>
+
+  <sky-demo-page-properties
+    sectionHeading="Toggle switch properties"
+  >
+    <sky-demo-page-property
+      propertyName="ariaLabel"
+      defaultValue="undefined"
+      [isOptional]="true"
+    >
+      Defines a string value to label the toggle switch and set the <sky-code>aria-label</sky-code> attribute to support <a stacheRouterLink="." fragment="accessibility">accessibility</a>. This property accepts <sky-code>string</sky-code> values.
```

Comment:
We link to an accessibility section that doesn't exist. Is that still coming?

Also, we set the default value to `undefined`. For other `ariaLabel` properties, we don't list a default value. Do we need to add `undefined` as the default value for those properties or remove it here?

Also, I know the wording in that first sentence corresponds to what we already have in place for other `ariaLabel` properties, but I wonder if we could tighten it up here and in those other components. The last sentence explains that this accepts string values, so would we lose anything by changing "Defines a string value to label the toggle switch..." to just "Labels the toggle switch..."? ... If we make that change here, we'd also need to change "set" to "sets." That said, in the other `ariaLabel` properties, I don't see explicit references to the `aria-label` attribute in the property descriptions. Should those properties also mention this? Or should we remove it here for consistency?

Finally, should this property make some reference to the `sky-toggle-switch-label` component? I assume so. Other components have `ariaLabel` and `ariaLabelledBy` properties that sorta work in conjunction. Since we don't have `ariaLabelledBy` here, I assume `sky-toggle-switch-label` is the default element to label the toggle switch and that you wouldn't need to label it and also use `ariaLabel`. But I'm not sure. it seems like we should mention `sky-toggle-switch-label` somewhere in the docs, so maybe this is a logical place.

Reply:
Should the accessibility link be to this section?

Reply:
Usually when we have links like this in `ariaLabel` properties, it points to an accessibility section within the same topic, like in the radio button topic at https://developer.blackbaud.com/skyux/components/radio.